src/dockerflow/logging.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, fmt=None, datefmt=None, style="%", logger_name="Dockerflow"):
         if sys.version_info[:2] < (3, 1):
             parent_init(self, format, datefmt)
         else:
-            parent_init(self, format, datefmt, style)
+            parent_init(self, datefmt=datefmt, style=style)
```

Comment:
```suggestion
            parent_init(self, fmt=format, datefmt=datefmt, style=style)
```

Reply:
Unless I misunderstand the stdlib code, I don't really agree with this. 

The `fmt` parameter isn't supposed to be a callable. Look at the default value on the default style https://github.com/python/cpython/blob/v3.8.0/Lib/logging/__init__.py#L413-L421

Reply:
And BTW what result do you expect by passing the `format` function?

Reply:
See https://github.com/python/cpython/pull/9703 and https://travis-ci.org/Kinto/kinto/jobs/599165067#L298

Reply:
Woops, you're right, I meant to pass the `fmt` variable, not the `format` variable.
```suggestion
            parent_init(self, fmt=fmt, datefmt=datefmt, style=style)
```

IIUC we need to keep the call signature of the `logging.Formatter.__init__`, and [it takes a `fmt` parameter](https://github.com/python/cpython/blob/v3.8.0/Lib/logging/__init__.py#L555).

In fact this seems to be a regression introduced in https://github.com/mozilla-services/python-dockerflow/pull/25/files#diff-792fa8a235db26fe95e4b30a7b16b8aeL50-R60! The parameter passed into `__init__` changed from format to fmt, but not the name of the variable passed to the parent init. 😬